uk.co.compucorp.civicrm.hrcore/backstop_data/casper_scripts/ssp-leave-absences/staff/my-report-manager-show-comments.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var page = require('../../../page-objects/ssp-leave-absences-my-leave-report');
+
+// precondition: need to have the second leave request on the pending list with
+// at least a comment from manager
+module.exports = function (casper) {
+  var myReport = page.init(casper);
+  myReport.openSection('\'pending\'');
+  myReport.openActions(2);
+
+  myReport.editRequest(2).then(function (modal) {
+    modal.selectTab('\'Comments\'');
```

Comment:
For both `selectTab` and `openSection` add the `\'` part in the function definition so that it can be removed when calling the same.